django-stubs/db/models/enums.pyi

```diff
@@ -21,6 +21,7 @@ else:
     class IntEnum(int, ReprEnum): ...
     class StrEnum(str, ReprEnum): ...
 
+@deprecated("ChoicesMeta is deprecated in favor of ChoicesType and will be removed in Django 6.0.")
```

Comment:
I'm not sure that this works either as `ChoicesType` is aliased to `ChoicesMeta` so will show the same deprecation warning.

As an aside, we should probably also invert the alias so that we have `class ChoicesType(EnumType)` and `ChoicesMeta = ChoicesType` for the alias.